src/Mod/Draft/WorkingPlane.py

```diff
@@ -861,6 +895,15 @@ def alignToSelection(self, offset=0):
                 "draft", "Selected Shapes must define a plane") + "\n")
             return False
 
+        # check for face + edge
+        if len(names) == 2:
+            if  any(["Face" in n for n in names]) and any(["Edge" in n for n in names]):
+                edge = shapes[[names.index(n) for n in names if n.startswith("Edge")][0]]
+                face = shapes[[names.index(n) for n in names if n.startswith("Face")][0]]
+                if edge.hashCode() in [e.hashCode() for e in face.Edges]:
+                    if super().align_to_face_and_edge(face, edge, offset):
+                        return True
+
```

Comment:
This is not wrong of course, but IMO it is preferable to not update this obsolete function.